package.json

```diff
@@ -24,7 +24,8 @@
 	},
 	"browserslist": [
 		"last 2 versions",
-		"not ie <= 11"
+		"not ie <= 11",
```

Comment:
No need for the other two entries anymore if extending from `@nextcloud/browserlist-config` :wink: